### PR TITLE
flameshot: add Chinese translation

### PR DESCRIPTION
**Pull request by jtm-lis**
kindly translated by Andrew.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/linux/flameshot.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+# 火焰弹
```
Comment (navarroaxel) [inline pages.zh/linux/flameshot.md:2]:
```suggestion
# flameshot
```

File: pages.zh/linux/flameshot.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+带有gui界面的Screenshot工具，支持基本的图像编辑，例如文本，形状，颜色和imgur。更多信息 https://flameshot.js.org
```
Comment (navarroaxel) [inline pages.zh/linux/flameshot.md:4]:
```suggestion
带有gui界面的Screenshot工具，支持基本的图像编辑，例如文本，形状，颜色和imgur。更多信息: <https://flameshot.js.org>
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @jtm-lis, I left some comments to follow the guidelines.

File: pages.zh/linux/flameshot.md (hunk 1 above)
Comment (jtm-lis) [reply pages.zh/linux/flameshot.md:2]: Okay thank you sorry about that!

File: pages.zh/linux/flameshot.md (hunk 2 above)
Comment (jtm-lis) [reply pages.zh/linux/flameshot.md]: Sorry I missed that. Thank you!

**Review by navarroaxel — APPROVED** [review]
Thanks @jtm-lis!

File: pages.zh/linux/flameshot.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# flameshot
+
+带有gui界面的Screenshot工具，支持基本的图像编辑，例如文本，形状，颜色和imgur。更多信息: <https://flameshot.js.org>
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:3]: Missing syntax here
```suggestion
> 带有gui界面的Screenshot工具，支持基本的图像编辑，例如文本，形状，颜色和imgur。更多信息: <https://flameshot.js.org>.
```

The `> ` indicates a blockquote. The full-stop at the end is required by the linter (unfortunately the linter was implemented before multiple languages were supported by tldr-pages).

File: pages.zh/linux/flameshot.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- 在GUI模式下启动Flameshot:
+
+flameshot launcher
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:7]: Missing code block syntax.
```suggestion
`flameshot launcher`
```

File: pages.zh/linux/flameshot.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+通过单击并拖动来截取屏幕截图:
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:9]: Missing `-` dash for a bullet point here
```suggestion
- 通过单击并拖动来截取屏幕截图:
```

File: pages.zh/linux/flameshot.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- flameshot gui
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:11]: Rogue dash here, and missing code block syntax.
```suggestion
`flameshot gui`
```

File: pages.zh/linux/flameshot.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+全屏截图:
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:13]: Same here.
```suggestion
- 全屏截图:
```

File: pages.zh/linux/flameshot.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+- flameshot full
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:15]: Same here.
```suggestion
`flameshot full`
```

File: pages.zh/linux/flameshot.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+将保存屏幕快照的路径设置为:
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:17]: Same here.
```suggestion
- 将保存屏幕快照的路径设置为:
```

File: pages.zh/linux/flameshot.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+- flameshot full --path {{path/to/directory}}
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:19]: Same here.
```suggestion
`flameshot full --path {{path/to/directory}}`
```

File: pages.zh/linux/flameshot.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+将屏幕截图延迟N毫秒，然后输出到剪贴板:
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:21]: Same here.
```suggestion
- 将屏幕截图延迟N毫秒，然后输出到剪贴板:
```

File: pages.zh/linux/flameshot.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+- flameshot full --delay {{2000}} --clipboard
```
Comment (sbrl) [inline pages.zh/linux/flameshot.md:23]: Perhaps this will read better?
```suggestion
`flameshot full --delay {{2000}} --clipboard`
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the translation, @jtm-lis! I've left some comments below for you to review.

Looks like some wires have been crossed with respect to the syntax here - I've left some comments below to correct the issues. The syntax for a tldr page is quite specific - try looking at the syntax in the English version of this page.

**Comment by jtm-lis** [comment]
@sbrl  Hey I feel really bad, I submitted this late at night before bed, and didn't figure I would of caused such strife.

Anyway next up is French, so that one will be done right.

**Comment by jtm-lis** [comment]
Reping @sbrl 

Please review the above, let me know if this reping is not needed and / or welcome. I figured you took a glance and left to more urgent pastures as often happens to me.

**Review by sbrl — APPROVED** [review]
Not a problem at all, @jtm-lis! I'm happy to help :-)

The extra ping doesn't actually have any effect - I get around to my GitHub notification once every few days :-)

Anyway, this looks fine to me now! Don't speak the language, ofo course :P

**Review by sbrl — APPROVED** [review]

**Pull request merged**